torch/_ops.py

```diff
@@ -96,7 +100,13 @@ def has_kernel_for_any_dispatch_key(self, ks):
                 return True
         return False
 
+    def fallthrough(self, dispatch_key):
+        self.non_fallthrough_keys = self.non_fallthrough_keys.remove(dispatch_key)
+
     def py_impl(self, k):
+        if isinstance(k, torch._C.DispatchKey) and not self.non_fallthrough_keys.has(k):
+            self.non_fallthrough_keys = self.non_fallthrough_keys.add(k)
+
```

Comment:
Under the suggestion above, the way to add a fallthrough is opoverload.py_impl(DispatchKey::Autograd, torch.library.fallthrough_kernel). Which is closer to how it's done in C++ TORCH_LIBRARY.